Theme fix

XinYenFon opened the pull request:
fixes #2611 but needs some tests.

Signed-off-by: Antes antes@simplemachines.org

Oldiesmann committed:
! Gender stats weren't being calculated properly (fixes #2666)

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

XinYenFon committed:
! move things to css and fix them... login popup centeralization (oh?)

Signed-off-by: Antes <antes@simplemachines.org>

XinYenFon committed:
meh stupid kitty...

Signed-off-by: Antes <antes@simplemachines.org>

bperry921 commented:
pardon the noob question, where would i look to determine if I have write access to merge? and if I do how/where to accomplish task, and by merging does it mean that i get the fix for my own version before waiting for the fix to be inserted to the full big picture, in other words, not sure what advantage the term merging gets ya? you say it needs testing, that's me man all the way, i'm pumped to help

XinYenFon commented:
If you know how to apply patch via Git (Bash), you can get this PR as patch and apply to your code.

https://github.com/SimpleMachines/SMF2.1/pull/2677.patch
https://github.com/SimpleMachines/SMF2.1/pull/2677.diff

Oldiesmann committed:
! Wrong variable in mention template (per #2678)

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

XinYenFon committed:
! Script echoed to guest user fixes #2682

Signed-off-by: Antes <antes@simplemachines.org>

live627 committed:
Update Who.php

Signed-off-by: John Rayes <live627@gmail.com>

live627 committed:
Update Who.php



Signed-off-by: John Rayes <live627@gmail.com>

Oldiesmann committed:
! Forgot password form was broken due to leftovers from when we had OpenID

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
+ Improve options for creating redirection topics (per #2642)

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
Wait until the last minute to handle deleting topic data, just to avoid any potential problems

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
Fix some undefined index errors and tweak a few other things

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
! Missed one...

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
And this should fix the "undefined index: is_approved" error...

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
Don't allow expiring redirects if the task to remove them is disabled
Set default expiration to one week instead of "never"

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
Add new setting on install/upgrade

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
More bugfixes:
- $reason wasn't defined
- createPost won't make an existing topic a redirection topic
- Incorrect link being used in the redirection topic
- Redirection topic wasn't being marked as read for the person who created it

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

Oldiesmann committed:
Typo fix

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

MissAllSunday committed:
Change the logic on loadCSSFile
allow schema less urls to be included

Signed-off-by: Suki <suki@missallsunday.com>

MissAllSunday committed:
Change the logic on loadJavascriptFile
Aloow schema less urls

Signed-off-by: Suki <suki@missallsunday.com>

MissAllSunday committed:
Change both instances to the new external param key

Signed-off-by: Suki <suki@missallsunday.com>

MissAllSunday committed:
Better wording

Signed-off-by: Suki <suki@missallsunday.com>

Oldiesmann committed:
Properly mark redirection topics as read when someone clicks on them...

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

live627 committed:
openID leftover

Signed-off-by: John Rayes <live627@gmail.com>

live627 committed:
Fix #2691

Restore lines removed in 2a8a2be3fdfaa33b67a00a3f599949dfa6bfc03c

Signed-off-by: John Rayes <live627@gmail.com>

Oldiesmann committed:
Slight language tweak to indicate that this also handles merged topic notices...

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

XinYenFon committed:
Thanks for the application we'll call you later... -script.js

Signed-off-by: Antes <antes@simplemachines.org>

XinYenFon committed:
! Move topic wasn't so responsive... we worried and take a look :P

Signed-off-by: Antes <antes@simplemachines.org>

XinYenFon committed:
Lets say improvement :P better to give some base ground to all devices (PC/Tablet/Phone)

Signed-off-by: Antes <antes@simplemachines.org>

XinYenFon committed:
! progress bar fixes and some little improvements.

Signed-off-by: Antes <antes@simplemachines.org>

live627 committed:
Fix #2697

Signed-off-by: John Rayes <live627@gmail.com>

Oldiesmann committed:
Tweak the topic maintenance a bit:
- Only include non-expiring redirection topics when deleting moved/merged notices (since we have a scheduled task to remove expiring ones)
- Don't delete moved/merged topic notices when deleting locked topics

Signed-off-by: Michael Eshom <oldiesmann@oldiesmann.us>

XinYenFon commented:
@Dragooon so we call this another rebase fail ? (Ref: http://git-scm.com/docs/git-rebase)
